lib/omniauth/strategies/qualtrics.rb

```diff
@@ -19,26 +18,31 @@ class Qualtrics < OmniAuth::Strategies::OAuth2
 
       info do
         {
-          "url" => access_token.client.site
+          'url' => access_token.client.site
         }
       end
 
+      def callback_url
+        full_host + script_name + callback_path
+      end
+
       # Override authorize_params so that we can be deliberate about the value for state
       # and not use the session which is unavailable inside of an iframe for some
       # browsers (ie Safari)
       def authorize_params
         # Only set state if it hasn't already been set
         options.authorize_params[:state] ||= SecureRandom.hex(24)
-        params = options.authorize_params.merge(options_for("authorize"))
+        options.authorize_params[:scope] = 'manage:all'
```

Comment:
This is not ideal--we'd rather it have just the scopes that we need, which you _should_ be able to do by setting them in the omniauth config...but I tried that and it didn't work. We can try to look into a better solution in the future.